tools/pom.xml

```diff
@@ -25,7 +23,7 @@
   <name>JBoss Fuse Tooling :: Tools</name>
 
   <modules>
-    <module>ide-catalog-generator</module>
+    <module>m2-repo-cleaner</module>
```

Comment:
the catalog generator isn't executed in the mvn build. it needs a manual exec:java call. Therefore we can skip that one too.